packages/calypso-products/src/plans-list.tsx

```diff
@@ -1844,6 +1919,39 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getPathSlug: () => 'ecommerce-2-years',
 	},
 
+	[ PLAN_ECOMMERCE_3_YEARS ]: {
+		...getPlanEcommerceDetails(),
+		term: TERM_TRIENNIALLY,
+		getBillingTimeFrame: WPComGetTriennialBillingTimeframe,
+		availableFor: ( plan ) =>
+			[
+				PLAN_FREE,
+				PLAN_WPCOM_STARTER,
+				PLAN_WPCOM_PRO,
+				PLAN_BLOGGER,
+				PLAN_BLOGGER_2_YEARS,
+				PLAN_PERSONAL_MONTHLY,
+				PLAN_PERSONAL,
+				PLAN_PERSONAL_2_YEARS,
+				PLAN_PERSONAL_3_YEARS,
+				PLAN_PREMIUM_MONTHLY,
+				PLAN_PREMIUM,
+				PLAN_PREMIUM_2_YEARS,
+				PLAN_PREMIUM_3_YEARS,
+				PLAN_BUSINESS_MONTHLY,
+				PLAN_BUSINESS,
+				PLAN_BUSINESS_2_YEARS,
+				PLAN_BUSINESS_3_YEARS,
+				PLAN_ECOMMERCE_MONTHLY,
+				PLAN_ECOMMERCE,
+				PLAN_ECOMMERCE_2_YEARS,
+				PLAN_WPCOM_PRO_MONTHLY,
```

Comment:
Nitpicking but it might be easier to comprehend what's going on if all Pro plans are together

Reply:
Cool! I'll sort that.